chemreg/substance/tests/test_views.py

```diff
@@ -478,6 +478,34 @@ def test_substance_patch(client, admin_user, substance_factory):
             assert resp.data[key]["id"] == model_dict[key]["id"]
 
 
+@pytest.mark.django_db
+def test_substance_patch_same_associated_compound(
+    client, admin_user, substance_factory
+):
+    client.force_authenticate(user=admin_user)
+    original_model = substance_factory.create(defined=True).instance
+    model_dict = substance_factory.build(illdefined=True).initial_data
+    model_dict.update(id="DTXSID205000001")
```

Comment:
This test works great but these two lines are unneeded.  Model dict is not used again within this test.

```suggestion
```